### PR TITLE
display queue length in queue info section

### PR DESCRIPTION
**Pull request by n-young**

### CODE REVIEW

**Review by nthnluu — CHANGES_REQUESTED** [review]

File: src/components/queue/QueueList/QueueList.tsx [hunk 1]
```diff
@@ -63,7 +62,7 @@ const QueueList: FC<QueueListProps> = ({queue, showCompletedTickets}) => {
                 <Typography variant="h6" fontWeight={600}>
                     Queue
                 </Typography>
-                {!queue.isCutOff && !queueEnded && !inQueue && !isTA(queue.course.id) &&
+                {!queue.isCutOff && !queueEnded && !inQueue && //!isTA(queue.course.id) &&
```
[the remark this answers is not in the document]
Comment (n-young) [reply src/components/queue/QueueList/QueueList.tsx:65]: fixed in most recent commit

File: src/pages/queue/[queueID].tsx [hunk 2]
```diff
@@ -29,9 +31,10 @@ export default function Queue() {
                 <Grid container spacing={4} marginTop={1}>
                     <QueueOptions queue={queue} queueID={queueID as string}
                                   showCompletedTickets={showCompletedTickets}
-                                  setShowCompletedTickets={setShowCompletedTickets}/>
-                    <QueueList queue={queue}
-                               showCompletedTickets={showCompletedTickets}/>
+                                  setShowCompletedTickets={setShowCompletedTickets}
```
[the remark this answers is not in the document]
Comment (n-young) [reply src/pages/queue/[queueID].tsx:34]: I kept the prop there so we can uncomment; it's only not being extracted in the params

File: src/pages/queue/[queueID].tsx [hunk 3]
```diff
@@ -7,12 +7,14 @@ import {toast} from "react-hot-toast";
 import AppLayout from "@components/shared/AppLayout";
 import QueueOptions from "@components/queue/QueueOptions";
 import QueueList from "@components/queue/QueueList";
+import {useTickets} from "@util/queue/hooks";
 
 export default function Queue() {
     const router = useRouter();
     const {queueID} = router.query;
     const [queue, queueLoading] = useQueue(queueID as string);
     const [showCompletedTickets, setShowCompletedTickets] = useState(true);
```
[the remark this answers is not in the document]
Comment (n-young) [reply src/pages/queue/[queueID].tsx:16]: Fixed

**Review by nthnluu — APPROVED** [review]

**Review by nthnluu — COMMENTED** [review]
lgtm

**Pull request merged**